src/client/remote_nodestream.ml

```diff
@@ -101,17 +101,18 @@ class remote_nodestream ((ic,oc) as conn) =
             loop_parts ()
           end
         | 3 ->
-          begin
-            Logger.debug_f_ "save_file" >>= fun () ->
-            Llio.input_string ic >>= fun name ->
-            Llio.input_int64 ic >>= fun length ->
-            Logger.info_f_ "got %s (%Li bytes)" name length >>= fun () ->
-            tlog_coll # save_tlog_file name length ic >>= fun () ->
-            loop_parts ()
-          end
+           begin
+             Logger.debug_f_ "save_file" >>= fun () ->
+             Llio.input_string ic >>= fun name ->
+             Sn.input_sn ic       >>= fun start_i ->
+             Llio.input_int64 ic  >>= fun length ->
+             Logger.info_f_ "retrieving %s (%Li bytes)" name length >>= fun () ->
+             tlog_coll # save_tlog_file name length ic >>= fun () ->
```

Comment:
`start_i` should be used here iso `name`
(also that means it's no longer needed to send `name` over the protocol)